packages/pipeline-editor/src/test/pipeline.spec.tsx

```diff
@@ -105,12 +105,13 @@ describe('@elyra/pipeline-editor', () => {
       });
 
       const browserFactory = { createFileBrowser, defaultBrowser, tracker };
+      const commands = new CommandRegistry();
```

Comment:
is there a reason you split this out?

Reply:
Oops, that's from an earlier change that I scrapped. I'll remove it.